Roslyakov Evgeny/oop.js

```diff
@@ -0,0 +1,266 @@
+//---------- Class MenuItem ----------//
+/*@constructor
+ *@param name       item name
+ *@param price      item price
+ *@param calories   item calories number
+ */
+function MenuItem(options) {
+    this.name = options.name;
+    this.price = options.price;
+    this.calories = options.calories;
+}
+
+/* Get the item name */
+MenuItem.prototype.getName = function () {
+    if(this.name === undefined) {
```

Comment:
here you can just check `if(!this.name)` checking the `undefined` value is a very rare case.

Reply:
yes, this even looks awkward for now) I've just fixed moments like this